src/@types/item/Holdable.ts

```diff
@@ -0,0 +1,38 @@
+import type { Pokemon } from "#app/field/pokemon";
+
+/**
+ * An item that can be held by a {@linkcode Pokemon}.
+ */
+export interface Holdable {
```

Comment:
```suggestion
export interface Holdable {
  /** The {@linkcode Pokemon} holding the item */
  holder: Pokemon;
```

I feel like this would help simplify the event contexts.

Reply:
That makes perfect sense :D